src/nimble.nim

```diff
@@ -615,15 +615,23 @@ proc listPaths(options: Options) =
       if kind != pcDir or not path.startsWith(options.getPkgsDir / name):
         continue
 
-      let
-        nimbleFile = path / name.addFileExt("nimble")
-        hasSpec = nimbleFile.existsFile
-
-      if hasSpec:
+      var nimbleFile = path / name.addFileExt("nimble")
+      var nimbleLinkTargetPath: string
+      if not nimbleFile.existsFile:
+        let nimbleLinkFile = path / name.addFileExt("nimble-link")
+        if fileExists(nimbleLinkFile):
+          let lns = readFile(nimbleLinkFile).splitLines()
+          nimbleFile = lns[0]
+          nimbleLinkTargetPath = lns[1]
```

Comment:
I'm fairly sure there is a function that deals with this logic, please use it instead of duplicating the code.